### ISSUE TITLE
3.x: Incorrect code example in Helidon SE Tracing doc

### ISSUE THREAD

**Issue by al3xandru** [helidon-io/helidon#6094]
In the Helidon SE Tracing doc https://helidon.io/docs/v3/#/se/tracing, the custom span creation code snippet is not correct for Helidon 3.x API:

```
Span span = request.tracer()
        .buildSpan("my-operation")
        .asChildOf(request.spanContext())
                         .start();
```

`io.helidon.webserver.Request#tracer()` returns an `io.helidon.tracing.Tracer` object which has a different API than the one used above.

**Comment by al3xandru** [comment]
Similar issue in https://helidon.io/docs/v3/#/se/guides/tracing

**Comment by ljamen** [comment]
@al3xandru I created a new PR for all of the Tracing updates. It is now #6113

**Comment by ljamen** [comment]
fixed by #6113 (approved / merged)
